src/blocking.rs

```diff
@@ -380,7 +381,26 @@ impl BlockingClient {
             ),
             None => format!("{}/scripthash/{:x}/txs", self.url, script_hash),
         };
-        Ok(self.agent.get(&url).call()?.into_json()?)
+        let resp = self.agent.get(&url).call(); //?.into_json();
+        match resp {
+            Ok(resp) => {
+                let resp_json = resp.into_json()?;
+                Ok(resp_json)
+            }
+            Err(ureq::Error::Status(code, resp)) => {
+                if is_status_too_many_requests(code) {
+                    eprintln!("Too many requests, sleeping for 500 ms");
+                    sleep(Duration::from_millis(500));
```

Comment:
this will cause a panic in wasm, might need to pass in a function that sleeps. for wasm sleep needs to be async too